### PR TITLE
Update setup.py

### PR DESCRIPTION
**Pull request by tsv19su254052**
corrected indents and hyphens

### CODE REVIEW

**Review by jvxram — APPROVED** [review]
Good

File: setup.py [hunk 1]
```diff
@@ -1,48 +1,42 @@
...
+               "Operating System :: POSIX :: Linux", 
+               "Operating System :: Microsoft :: Windows", 
+               'Programming Language :: Python', 
+               'Programming Language :: Python :: 3', 
+               'Programming Language :: Python :: 3.6', 
+               'Programming Language :: Python :: 3.7', 
+               'Programming Language :: Python :: 3.8', 
+               'Programming Language :: Python :: 3.9', 
+               'Framework :: Django :: 3.2', 
+               'Framework :: Django :: 4.0', ]
```
Comment (tsv19su) [inline setup.py:20]: good

File: setup.py [hunk 2]
```diff
@@ -1,48 +1,42 @@
...
+      url='https://github.com/microsoft/mssql-django', 
+      project_urls={'Release Notes': 'https://github.com/microsoft/mssql-django/releases', }, 
+      license='BSD', 
+      packages=find_packages(), 
+      install_requires=['django>=2.2,<4.1', 
+                        'pyodbc>=3.0', 
+                        'pytz', ], 
+      package_data={'mssql': ['regex_clr.dll']}, 
+      classifiers=CLASSIFIERS, 
+      keywords='django', )
```
Comment (tsv19su) [inline setup.py:42]: good

**Review by tsv19su — APPROVED** [review]
Good

**Comment by jmah8** [comment]
Hi, thanks for contributing but I am going to close this PR because we would like to keep the formatting of this file consistent with all the other files.

**Pull request closed without merging**